kafkabp/config.go

```diff
@@ -99,6 +100,12 @@ type ConsumerConfig struct {
 	// or it might make things worse.
 	// You are advised to test before using non-empty rack id in production.
 	RackID RackIDFunc `yaml:"rackID"`
+
+	// Optional. Defaults to 5. Values are in seconds.
+	//
+	// This affects how often the kafka consumer writes its offsets back to
+	// the cluster.
+	AutoCommitInterval int `yaml:"autoCommitInterval"`
```

Comment:
Can we use `time.Duration` directly here?

Reply:
great call, done. I barely know go, so the tests are for my own confidence more than anything, haha!